WMF Framework/Fetcher.swift

```diff
@@ -111,6 +111,19 @@ open class Fetcher: NSObject {
         return task
     }
     
+    @discardableResult public func performMobileAppsServicesGET<T: Decodable>(for URL: URL?, pathComponents: [String], priority: Float = URLSessionTask.defaultPriority, cancellationKey: CancellationKey? = nil, completionHandler: @escaping (_ result: T?, _ response: URLResponse?,  _ error: Error?) -> Swift.Void) -> CancellationKey? {
+        
+        //The accept profile is case sensitive https://gerrit.wikimedia.org/r/#/c/356429/
+        let headers = ["Accept": "application/json; charset=utf-8; profile=\"https://www.mediawiki.org/wiki/Specs/Summary/1.1.2\""]
+        let taskURL = configuration.wikipediaMobileAppsServicesAPIURLComponentsForHost(URL?.host, appending: pathComponents).url
+        let key = cancellationKey ?? UUID().uuidString
+        let task = session.jsonDecodableTask(with: taskURL, headers: headers, priority: priority) { (result: T?, response: URLResponse?, error: Error?) in
+            completionHandler(result, response, error)
```

Comment:
Should this task be untracked in `jsonDecodableTask`'s completion?

Reply:
@nambatee This is fixed, great catch!